dali/pipeline/executor/executor2/exec_graph_test.cc

```diff
@@ -397,6 +397,86 @@ TEST(ExecGraphTest, Exception) {
   }
 }
 
+TEST(ExecGraphTest, LoweredStructureMatch) {
+  graph::OpGraph def = GetTestGraph1();
+  ExecGraph g;
+  g.Lower(def);
+  ASSERT_EQ(g.Nodes().size(), def.OpNodes().size() + 1);
+  EXPECT_TRUE(g.Nodes().back().is_pipeline_output);
+  EXPECT_EQ(g.Nodes().back().inputs.size(), 2_uz);
+  auto def_it = def.OpNodes().begin();
+  auto ex_it = g.Nodes().begin();
+  for (; def_it != def.OpNodes().end(); def_it++, ex_it++) {
```

Comment:
Nitpick: skipping the first expression in `for` and using a coma in the last one makes this line quite hard to read :/